src/App.java

```diff
@@ -14,17 +14,126 @@ public void start() {
         printInstructions();
         doMadlib();
     }
-    
+
     /**
-     * Document what this method does here!
+     * This intruction tells you what you are supposed to do.
      */
     public void printInstructions() {
```

Comment:
It's strange that you put your instructions in a different method, not the one provided for that purpose